Description: Add "unicode" hack from the unicode package
 Just added here as a convenience.
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=560184
Author: Joachim Breitner <nomeata@debian.org>

Index: xscreensaver-git/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver-git.orig/driver/XScreenSaver.ad.in
+++ xscreensaver-git/driver/XScreenSaver.ad.in
@@ -356,6 +356,7 @@ XScreenSaver.bourneShell:		/bin/sh
 				goop --root				    \n\
 				grav --root				    \n\
 				ifs --root				    \n\
+				unicode --root				    \n\
 @GL_KLUDGE@ GL: 				jigsaw --root				    \n\
 				julia --root				    \n\
 -				kaleidescope --root			    \n\
